tests/test_assemblies.py

```diff
@@ -481,4 +485,64 @@ def test_fields_present_abnormal_sets(self, identifier, field):
     ])
     def test_fields_present_cue_conflict(self, identifier, field):
         assembly = brainscore.get_assembly(f"brendel.Geirhos2021_{identifier}")
-        assert hasattr(assembly, field)
\ No newline at end of file
+        assert hasattr(assembly, field)
+
+@pytest.mark.private_access
+class TestBaker2022:
+
+    # tests alignments that are the same across normal and inverted assemblies
+    @pytest.mark.parametrize('identifier, length', [
+        ('normal', 3702),
+        ('inverted', 4320),
+    ])
+    def test_stimulus_set_assembly_alignment(self, identifier, length):
+        full_name = f'Baker2022_{identifier}_distortion'
+        assembly = brainscore.get_assembly(full_name)
+        assert assembly.stimulus_set is not None
+        assert assembly.stimulus_set.identifier == f'Baker2022_{identifier}_distortion'
+        assert set(assembly.stimulus_set["animal"]) == set(assembly["truth"].values)
+        assert set(assembly.stimulus_set["stimulus_id"]) == set(assembly["stimulus_id"].values)
+        assert len(assembly.presentation) == length
+
+    # tests counts that are the same across normal and inverted assemblies
+    @pytest.mark.parametrize('identifier', [
+        'normal',
+        'inverted',
+    ])
+    def test_same_counts(self, identifier):
+        full_name = f'Baker2022_{identifier}_distortion'
+        assembly = brainscore.get_assembly(full_name)
+        assert len({x for x in set((assembly["truth"]).values) if x==x}) == 9
+        assert len(set((assembly["correct"]).values)) == 2
+
+    # tests number of subjects
+    @pytest.mark.parametrize('identifier, num_subjects', [
+        ('normal', 32),
+        ('inverted', 12),
+    ])
+    def test_subjects(self, identifier, num_subjects):
+        full_name = f'Baker2022_{identifier}_distortion'
+        assembly = brainscore.get_assembly(full_name)
+        assert len(set((assembly["subject"]).values)) == num_subjects
+
+    # tests number of configurations in {whole, fragmented, frankenstein} Inverted does not have fragmented)
+    @pytest.mark.parametrize('identifier, num_configs', [
+        ('normal', 3),
+        ('inverted', 2),
+    ])
+    def test_configs(self, identifier, num_configs):
+        full_name = f'Baker2022_{identifier}_distortion'
+        assembly = brainscore.get_assembly(full_name)
+        assert len(set((assembly["condition"]).values)) == num_configs
```

Comment:
```suggestion
    @pytest.mark.parametrize('identifier, num_configs', [
        ('normal', {'whole', 'fragmented', 'frankenstein'}),
        ('inverted', {'whole', 'fragmented'}),
    ])
    def test_conditions(self, identifier, conditions):
        full_name = f'Baker2022_{identifier}_distortion'
        assembly = brainscore.get_assembly(full_name)
        assert set((assembly["condition"]).values) == conditions
```
double-check if the values (`{'whole', 'fragmented', 'frankenstein'}`) are correct